### PR TITLE
Pending member

### PR DESCRIPTION
**Pull request by CristianAvalos**

### CODE REVIEW

**Review by LukeY23 — APPROVED** [review]
I think it looks good besides the tests failing

File: app/models/user.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    #relationships
+    # relationships
     belongs_to :role 
     has_one :storage_key
     has_many :rsvps
     
     validates :first_name, presence: true, allow_blank: false
     validates :last_name, presence: true, allow_blank: false
-    validates :email, presence: true, allow_blank: false
+    validates_uniqueness_of :email, presence: true, allow_blank: false
```
Comment (StevenGrimshaw) [inline app/models/user.rb:9]: This is amazing

File: app/views/dashboards/show.html.erb [hunk 2]
```diff
@@ -15,8 +15,9 @@
     <%= link_to 'Storage Keys', storage_keys_path %><br>
 
     <%= link_to "Sign Out", destroy_admin_session_path %>
-<% else %>
+<% elsif session[:user_role] == 'Pending' %>
```
Comment (StevenGrimshaw) [inline app/views/dashboards/show.html.erb:18]: Is it possible to have a user that isn't pending, member, or admin?

**Review by StevenGrimshaw — APPROVED** [review]
Looked over test failures and it looks like it is just because of the styling.

**Pull request merged**